### PR TITLE
Update InspectingResults.html

### PR DESCRIPTION
**Pull request by 0seastar0**
Spelling and grammar edits

### CODE REVIEW

File: docs/InspectingResults.html [hunk 1]
```diff
@@ -138,10 +137,10 @@ <h2>Using a browser to explore assembly results</h2>
 
 <h2 id=AccessControl>Access control</h2>
 <p>
-By default, the server runs in a mode in which it only responds
+By default, the server runs in a mode that it only responds
```
[the remark this answers is not in the document]
Comment (bkmartinjr) [reply docs/InspectingResults.html:140]: I would have said "...a mode where it only ..." or "... in which...".  I agree that use of "that" is unusual and doesn't read right to me.

A full rewrite is my preference:   "By default, the server responds to requests originating from the same computer in which the server is running."

Additional point: how do you restrict to the same user?  I wasn't aware that was even possible with the loopback, short of adding actual identity/authentication to the system

**Comment by paoloczi** [comment]
I will leave it for @0seastar0 to decide what the wording should be.

Regarding checking if it is the same user, for each incoming connection I use the `lsof` linux command to find the open socket on 127.0.0.1 and the port used by the Shasta server. The `lsof` command also returns the name of the user that owns the socket. If not the same as the user running the server, the incoming connection is closed. I don't claim this mechanism to be "secure" or elegant but it probably comes close enough for our purposes.

See `HttpServer::isLocalConnectionSameUser` in `shasta/src/HttpServer.cpp` (near line 659) for details.

**Comment by paoloczi** [comment]
To clarify - `--exploreAccess user` accepts only local connections  (127.0.0.1) from the same user. In that mode, remote connections are not accepted, so no check about remote users needs to happen.

File: docs/InspectingResults.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (0seastar0) [reply docs/InspectingResults.html:140]: "By default, the server responds to requests originating from the same computer the server is running on."

[the remark this answers is not in the document]
Comment (paoloczi) [reply docs/InspectingResults.html:140]: Sounds good to me, go ahead with that and I will merge the PR.

[the remark this answers is not in the document]
Comment (paoloczi) [reply docs/InspectingResults.html:140]: Actually, that statement is incorrect. The default for `--exploreAccess` is `user`, which means by default we only accept requests from the same computer **and** the same user. The original wording before this PR stated this correctly.

**Pull request merged**